craft_application/application.py

```diff
@@ -894,6 +909,18 @@ def _initialize_craft_parts(self) -> None:
         self._enable_craft_parts_features()
         self._register_default_plugins()
 
+    def _run_tests(self) -> None:
```

Comment:
Ideally this should be in the Test command

Reply:
To be addressed as part of the execution refactoring.